lib/workos/client.rb

```diff
@@ -107,7 +110,7 @@ def handle_error_response(response:)
           request_id: response['x-request-id'],
         )
       when 404
-        raise APIError.new(
+        raise NotFoundError.new(
```

Comment:
ahh, these more specific error classes are nice. i guess this will be a breaking change though? (needs a major version bump?).

Reply:
Yep afraid so, but it's okay as we have some other changes that are ready to go and require a major version.